registry/storage/registry.go

```diff
@@ -26,6 +26,10 @@ func NewRegistryWithDriver(ctx context.Context, driver storagedriver.StorageDriv
 	var statter distribution.BlobStatter = &blobStatter{
 		driver: driver,
 		pm:     defaultPathMapper,
+		tomb: tomb{
```

Comment:
Tombstone on global `blobStatter` makes no sense since the tombstones are repository unique. It is also not used by the `blobStatter`.